package.json

```diff
@@ -13,6 +13,8 @@
     "react-dom": "^16.9.0",
     "react-router-dom": "^5.0.1",
     "react-scripts": "3.1.1",
+    "react-select": "^3.0.8",
+    "react-window": "^1.8.5",
```

Comment:
Yeah I am aware of it that. It because these libraries was in gitlap dev but I forget the merge into github dev. We can keep it these to library. I will be using it on my `selectizeDropdown Feature`.